api/tables/index.js

```diff
@@ -1,45 +1,10 @@
-/** @typedef {import('@aws-sdk/client-dynamodb').CreateTableCommandInput} CreateTableCommandInput */
 /** @typedef {import('@serverless-stack/resources').TableProps} TableProps */
 
-/**
- * Convert SST TableProps to DynamoDB `CreateTableCommandInput` config
- *
- * @param {TableProps} props
- * @returns {Pick<CreateTableCommandInput, 'AttributeDefinitions' | 'KeySchema'>}
- */
-export function dynamoDBTableConfig ({ fields, primaryIndex }) {
-  if (!primaryIndex || !fields) throw new Error('Expected primaryIndex and fields on TableProps')
-  const attributes = Object.values(primaryIndex)
-  const AttributeDefinitions = Object.entries(fields)
-    .filter(([k]) => attributes.includes(k)) // 'The number of attributes in key schema must match the number of attributes defined in attribute definitions'
-    .map(([k, v]) => ({
-      AttributeName: k,
-      AttributeType: v[0].toUpperCase()
-    }))
-  const KeySchema = [
-    { AttributeName: primaryIndex.partitionKey, KeyType: 'HASH' }
-  ]
-  if (primaryIndex.sortKey) {
-    KeySchema.push(
-      { AttributeName: primaryIndex.sortKey, KeyType: 'RANGE' }
-    )
-  }
-  return {
-    AttributeDefinitions,
-    KeySchema
-  }
-}
-
 /** @type TableProps */
 export const storeTableProps = {
   fields: {
     space: 'string',        // `did:key:space`
     link: 'string',         // `bagy...1`
-    size: 'number',         // `101`
-    origin: 'string',       // `bagy...0` (prev CAR CID. optional)
-    issuer: 'string',       // `did:key:agent` (issuer of ucan)
-    invocation: 'string',   // `baf...ucan` (CID of invcation UCAN)
-    insertedAt: 'string',   // `2022-12-24T...`
```

Comment:
nit: any reason to remove? I understand it is optional, but I actually would like to have a place where we can see the schema.